bashunit

```diff
@@ -40,6 +40,12 @@ while [[ $# -gt 0 ]]; do
       SIMPLE_OUTPUT=false
       shift
       ;;
+    --env)
+      # shellcheck disable=SC1090
+      source "$2"
+      shift
+      shift
+      ;;
```

Comment:
why 2 `shift`?

Reply:
1st) for the -e|--env
2nd) for the value itself
> **Note**: You can see the filter using the same strategy.